app/api/events/upcoming/route.ts

```diff
@@ -14,8 +14,7 @@ export async function GET(req: NextRequest) {
     status: 200,
     headers: {
       "Content-Type": "application/json",
-      "Access-Control-Allow-Origin":
-        "https://sapeoplestagjthgmptzb46i.z8.web.core.windows.net",
+      "Access-Control-Allow-Origin": "*",
```

Comment:
Don’t use *
Add the actual urls

Reply:
Done - I've updated the endpoints to return the appropriate access-control-allow-origin header only when the domain is trusted